pages/register.tsx

```diff
@@ -145,515 +72,489 @@ export default function Register() {
-              />
-              <text className="pl-2">Capital One</text>
-            </label>
-            <label>
-              <br />
-              <input
-                className="form-checkbox h-5 w-5"
-                name="EB"
-                type="checkbox"
-                checked={registrationData.companies.includes('EB')}
-                onChange={(e) => updateCompanies(e)}
-              />
-              <text className="pl-2">Ebay</text>
-            </label>
-            <label>
-              <br />
-              <input
-                className="form-checkbox h-5 w-5"
-                name="FB"
-                type="checkbox"
-                checked={registrationData.companies.includes('FB')}
-                onChange={(e) => updateCompanies(e)}
-              />
-              <text className="pl-2">Facebook</text>
-              <br />
-              <br />
-            </label>
-
-            <label>
-              Upload your resume:
-              <br />
-              <input
-                onChange={(e) => setResumeFile(e.target.files[0])}
-                name="resume"
-                type="file"
-                formEncType="multipart/form-data"
-              />
-              <br />
-            </label>
-            <br />
-            <button
-              type="button"
-              className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 border border-blue-700 rounded "
-              onClick={() => handleSubmit()}
-            >
-              Submit
-            </button>
-          </form>
-        </div>
+
+              <label htmlFor="companies" className="mt-4">
+                Companies to send my resume to:
+              </label>
+              <div role="group" aria-labelledby="checkbox-group" className="flex flex-col">
+                <label>
+                  <Field type="checkbox" name="companies" value="State Farm" />
+                  &nbsp;State Farm
+                </label>
+                <label>
+                  <Field type="checkbox" name="companies" value="American Airlines" />
+                  &nbsp;American Airlines
+                </label>
+                <label>
+                  <Field type="checkbox" name="companies" value="Capital One" />
+                  &nbsp;Capital One
+                </label>
+                <label>
+                  <Field type="checkbox" name="companies" value="Ebay" />
+                  &nbsp;Ebay
+                </label>
+                <label>
+                  <Field type="checkbox" name="companies" value="Facebook" />
+                  &nbsp;Facebook
+                </label>
+              </div>
+
+              <label className="mt-4">
+                Upload your resume:
+                <br />
+                <input
+                  onChange={(e) => setResumeFile(e.target.files[0])}
+                  name="resume"
+                  type="file"
```

Comment:
file also accepts random file formats that should not be accepted (.java and .zip were the ones tested)